src/main/java/org/victorrobotics/frc/dtlib/actuator/motor/ctre/DTTalonFX.java

```diff
@@ -197,7 +206,7 @@ public double getEncoderPosition() {
 
     @Override
     public double getEncoderVelocity() {
-        return internal.getSelectedSensorVelocity() / TICKS_PER_REV * 10;
+        return internal.getSelectedSensorVelocity() / TICKS_PER_REV * 10 * SECONDS_PER_MINUTE;
```

Comment:
This returns the velocity in RPM, which is an important capability. Consider renaming this to something like `getMotorRPM` or `getRPM`.

Reply:
How about `getEncoderVelocityRPM` or `getVelocityRPM`? I don’t think `getRPM` is specific enough (could just be commanded velocity), and the word “motor” shouldn’t appear in a `DTMotor` method.